api/src/main/java/io/opentelemetry/metrics/Counter.java

```diff
@@ -40,17 +40,6 @@
     @Override
     Builder setConstantLabels(Map<String, String> constantLabels);
 
-    /**
```

Comment:
Is this Counter.Builder interface even used any more?

Reply:
In fact, is the `Counter` interface even used?

Reply:
Sorry, my bad, I forgot to delete it :)